atmos.yaml

```diff
@@ -76,6 +76,16 @@ stacks:
     - "**/_defaults.yaml"
   # Can also be set using 'ATMOS_STACKS_NAME_PATTERN' ENV var
   name_pattern: "{tenant}-{environment}-{stage}"
+  list:
+    columns:
+      - name: Stack
+        value: '{{ .atmos_stack }}'
+      - name: Tenant
+        value: '{{ index .vars "tenant" }}'
```

Comment:
In documentation, explain why we should use "index" function, so we don't error on undefined fields.